templates/includes/card.html

```diff
@@ -167,12 +169,14 @@ <h3>Demos</h3>
   </div>
 </li>
 {% endmacro %} {% macro timelineCard(api, releases, versions, loop) %}
-<li class="card__holder" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}" style="{{api | timelinePosition(releases)}}; grid-row: {{loop.index + 1}};">
+<li class="card__holder" data-title="{{api.title}}" data-shipping="{{api.shipping | dump}}" data-type="{{type}}" data-bug="{{api.id}}" data-where="{{api.where | dump | lower}}" data-pwa="{{api.pwa}}" data-iwa="{{api.iwa}}" style="{{api | timelinePosition(releases)}}; grid-row: {{loop.index + 1}};">
   <div class="card">
     <div class="card--header">
       <button class="card--more card--title">
         <h2 class="card--title">
-          {% if api.pwa %}
+          {% if api.iwa %}
+          <img class="card--location" src="/images/iwa.svg" alt="IWA" title="IWA" />
+          {% elseif api.pwa %}
```

Comment:
Not sure again about the "elseif". WDYT?

Reply:
`elseif` is OK: https://mozilla.github.io/nunjucks/templating.html#:~:text=elif%20(or-,elseif,-%2C%20which%20is%20simply, but `elif` seems more common. Let me `elif` it all.